core/build.gradle

```diff
@@ -2,17 +2,17 @@ apply plugin: 'com.android.library'
 apply plugin: 'bintray-release'
 
 dependencies {
-    compile 'com.android.support:support-v4:19.1.0'
+    compile 'com.android.support:support-v4:23.0.1'
 }
 
 android {
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 19
+        targetSdkVersion 22
```

Comment:
The target on the demo is 23 I just noticed :x: Is there downside to making it 23?